### PR TITLE
new icon: readthedocs (original, original-wordmark, line)

### PR DESCRIPTION
**Pull request by ConX**
## Double-check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://read-the-docs-guidelines.readthedocs-hosted.com/branding.html#logo-guidelines

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7335,6 +7335,37 @@
             }
         ]
     },
+    {
+        "name": "readthedocs",
+        "altnames": [
+            "read the docs",
```
Comment (lunatic-fox) [inline devicon.json:7341]: Maybe we can keep only "Read the Docs" here. I think we can use lowercase in search engine if it's not already. 🤔

Comment (Snailedlt) [reply devicon.json:7341]: @ConX For quick application if you agree with lunatic :)
```suggestion
```

**Review by Snailedlt — COMMENTED** [review]
Other than what @lunatic-fox already commented, this looks good to me :)

File: devicon.json (hunk 1 above)
Comment (Panquesito7) [reply devicon.json:7341]: I agree with this. 👍

File: devicon.json [hunk 2]
```diff
@@ -7335,6 +7335,39 @@
...
+            "Read The Docs"
+        ],
+        "tags": [
+            "documentation"
```
Comment (Panquesito7) [inline devicon.json:7345]:
```suggestion
            "documentation",
            "python",
            "open-source"
```

**Comment by ConX** [comment]
@Snailedlt, this should be ready to merge

**Review by lunatic-fox — APPROVED** [review]
Looks good to me! ✔
Thank you for this contribution! 🚀

**Review by Snailedlt — APPROVED** [review]
Great :heavy_check_mark: 
Thank you for another great contribution! :100:

**Comment by Snailedlt** [comment]
@ConX @lunatic-fox any idea why the optimize icons check is failing?

**Comment by lunatic-fox** [comment]
> @ConX @lunatic-fox any idea why the optimize icons check is failing?

I was about to comment it later. 😄
I think is something related to repository permissions, because the exit 128 response when the bot tries to commit.
I think if is something that's not possible to fix now, I should probably make a PR to remove this feature at moment.

**Comment by Snailedlt** [comment]
@lunatic-fox aha! So probably the same issue as with the in-develop labeler then? See #1410

**Comment by Snailedlt** [comment]
Anyways, this should be safe to merge now, right @lunatic-fox ?

**Comment by lunatic-fox** [comment]
> Anyways, this should be safe to merge now, right @lunatic-fox ?
Sure! 👍🏼

**Pull request merged**